metadata/launchdarkly_js-core.json

```diff
@@ -8,7 +8,8 @@
       "languages": ["JavaScript", "TypeScript"],
       "releases": {
         "tag-prefix": "akamai-server-base-sdk-"
-      }
+      },
+      "userAgents": ["AkamaiEdgeSDK"]
```

Comment:
This AkamaiEdgeSDK being used for both is going to potentially cause some minor inconvenience.